src/components/victory-line.jsx

```diff
@@ -1,28 +1,16 @@
 import flatten from "lodash/array/flatten";
 import last from "lodash/array/last";
-import take from "lodash/array/take";
-import union from "lodash/array/union";
-import zip from "lodash/array/zip";
-import pluck from "lodash/collection/pluck";
 import sortBy from "lodash/collection/sortBy";
-import isArray from "lodash/lang/isArray";
 import isEmpty from "lodash/lang/isEmpty";
-import isFunction from "lodash/lang/isFunction";
 import isNull from "lodash/lang/isNull";
-import isNumber from "lodash/lang/isNumber";
-import isObject from "lodash/lang/isObject";
 import isUndefined from "lodash/lang/isUndefined";
 import merge from "lodash/object/merge";
 import pick from "lodash/object/pick";
-import max from "lodash/math/max";
-import min from "lodash/math/min";
-import lodashRange from "lodash/utility/range";
-import React, { PropTypes } from "react";
+import React from "react";
```

Comment:
wouldn't it be better to keep:

```jsx
import React, { PropTypes } from 'react';
```

and then have something like `UtilsPropTypes` in the `victory-util`:

```jsx
import { Chart, Data, Domain, UtilsPropTypes, Scale } from 'victory-util';
```

usually when I see `PropTypes` I would think of `React.PropTypes`